src/analyses/threadId.ml

```diff
@@ -32,24 +32,30 @@ struct
   module N = Lattice.Flat (VNI) (struct let bot_name = "unknown node" let top_name = "unknown node" end)
   module TD = Thread.D
 
-  module D = Lattice.Prod3 (N) (ThreadLifted) (TD)
+  module D = Lattice.Prod3 (N) (ThreadLifted) (Lattice.Prod(TD)(TD))
```

Comment:
Should document here the meaning of different components because the difference between the two `TD`s is not clear.